activity.py

```diff
@@ -161,6 +171,13 @@ def _easier_button_cb(self, button):
     def _harder_button_cb(self, button):
         self.game.harder()
 
+    def set_mode(self, mode):
+        self._mode_button.set_active(mode)
+        self._mode_button.connect('toggled', self._mode_button_cb)
+
+    def _mode_button_cb(self, button):
+        self.game.mode(int(button.get_active()))
```

Comment:
Why convert to an int here?

Reply:
It could work fine with `bool` as well, will fix it.